index.html

```diff
@@ -142,12 +142,12 @@ <h2>About Myself</h2>
                 <h2>Contact me</h2>
                 <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
                 <form action="https://formspree.io/f/myyvzkjz" method="POST" class="form-input">
-                <div class="input-field error">
+                <div class="input-field">
                   <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" aria-labelledby="name" required>
                 </div>
-                  <div class="input-field success">
+                  <div class="input-field">
                     <input type="email" name="Email" id="email" placeholder="Enter your email" aria-labelledby="email" required>
-                    <small style="color: red; font-size: 30px;"></small>
+                    <span class="error"></span>
```

Comment:
- [ ] Following the project requirements, the error message should show up close to the "submit" button.